### ISSUE TITLE
Run/launch shortcut with gestures

### ISSUE THREAD

**Issue by victor0d** [TBog/TBLauncher#257]
The feature **"Run app/shortcut" gesture** https://github.com/TBog/TBLauncher/issues/113 closed already, allows to launch apps through gestures. And works flawlessly. Would if be possible to add shortcuts too. The main reason is to launch scripts.

**Comment by TBog** [comment]
> The main reason is to launch scripts.

Are the "scripts" something we should add to the launcher?

**Comment by daffydock** [comment]
@victor0d 

Agreed. What kind of scripts are you referring about? One can technically already run Termux and Tasker scripts via regular shortcuts and those can be very powerful. I think you can run PhoneProfilePlus profiles changes via shortcuts, too.

**Comment by TBog** [comment]
I can easily add a way to launch intents, but I'm sure the interface for making intents with extra data is going to be ugly.
I'm thinking intents could be called shortcuts in the launcher and function the same way.

**Comment by victor0d** [comment]
@ltGuillaume @daffydock : I completely agree that Termux and Tasker are capable of scripting most of the things. And in general enough for myself.

My initial request was simply to add the capability to launch shortcuts besides apps within the launcher actions. THANKS TBog for the proposed solution :+1: 

Nevertheless, TBog question (_Are the "scripts" something we should add to the launcher?_) is quite interesting from the discussion point of view.

I understand, that it could become big and complex (something to avoid), but it opens the possibility to add functionality that is not possible with Termux/Tasker only, and maybe to hard to support in the long run if embedded in the launcher itself.

To make it simpler, I guess it may be easier to discuss usage cases. Some ideas that come to my mind at this point (for discussion purposes only) could be things like:
- Interfacing with additional search providers (and present the output in the results list):
. File managers: to search for files 
. Docs/Notes apps: to search for text
. Music players: to search for songs/albums/...
. Messaging apps: to search for messages or contacts
. ...
- Integrating background apps: changing the gestures depending on the launched app (e.g. when playing music, gestures could temporarily reassigned to interact with songs/volume)
- adding/deleting freeze/defrost shortcuts automatically when interacting with freezing apps or island/shelter hibernating apps.

Anyhow, this has really become much more complex than the original request :0)

**Comment by daffydock** [comment]
@TBog 

Albeit I do not use them as often, I would not mind this option. I am sure many people use Intents. Personally I use them, I leverage two apps, Activity Manager and Activity Launcher.  Both on Github. 

Perhaps, if it helps, you may perhaps take a look at those apps, maybe you might be able to find something useful from them. Whatever you decide, thank you, either way.

**Comment by victor0d** [comment]
@TBog It works now in the release: [v6.10-dev.1](https://github.com/TBog/TBLauncher/releases/tag/v6.10-dev.1) - THANKS TBog.

@daffydock thank you for suggesting T-E-L. I was not aware of it. It is a very interesting project. Having a tiling WM is quite cool. It is though, quite power hungry.
